spirv_glsl.hpp

```diff
@@ -66,6 +66,7 @@ class CompilerGLSL : public Compiler
 		// If true, Vulkan GLSL features are used instead of GL-compatible features.
 		// Mostly useful for debugging SPIR-V files.
 		bool vulkan_semantics = false;
+		uint8_t pad0[3];
```

Comment:
From my perspective the result of fixing this warning is actively harmful to the codebase - it generates no value since SPIRV-Cross doesn't care about memory consumption (and even if it did, structs like this one just don't matter period), and changes to the code will change the amount of required padding which would invalidate the fixes and/or make the struct layout even worse compared to what it would have been without the padding fields.